MetaMask/metamask-extension
# Fix Actionable message for new Storybook format

Updating `ActionableMessage` story:
- Add js doc comments to proptypes to allow ArgsTable to show up
- Add `README.MDX`
- Add controls for interaction of component

**Manual testing steps**
- Go to latest CI build of storybook or run `yarn storybook`
- Search "ActionableMessage" in storybook
- Use Controls to interact with component
- Check docs page to see Props table

**Images**
<img width="1437" alt="Screen Shot 2021-12-02 at 3 15 22 PM" src="https://user-images.githubusercontent.com/8112138/144518438-df389102-3965-491f-ae9f-364f720e084c.png">

![screencapture-localhost-6006-2021-12-02-15_17_07](https://user-images.githubusercontent.com/8112138/144518462-47ca5a6b-a838-4ea1-864b-d7cc6f078ff7.png)
